### PR TITLE
Create E2E testing for seek actions

### PR DESCRIPTION
**Pull request by rodrigo-rodrigues-ctw**
- create seek buttons e2e tests

## User-Facing Changes

N/A

## Description

Create E2E testing for seek actions

## Checklist

- [x] The web version was tested and it is running ok
- [x] The desktop version was tested and it is running ok
- [x] This change is covered by unit tests
- [x] Files constants.ts, types.ts and *.style.ts have been checked and relevant code snippets have been relocated

### CODE REVIEW

File: e2e/tests/desktop/player/playback-seek.desktop.spec.ts [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+async function clickPlayblackSlider(mainWindow: Page, fraction: number) {
+  const slider = mainWindow.getByTestId("playback-slider");
+  const box = await slider.boundingBox();
+  if (!box) {
+    throw new Error("Slider bounding box not found");
+  }
+
+  // Add small offsets for edge cases to ensure click is within bounds
+  const offset = 2; // pixels from edge
+  const safeX =
```
Comment (luluiz) [inline e2e/tests/desktop/player/playback-seek.desktop.spec.ts]: Avoid sub ternary conditions.
You can simplify this expression for better readability and make your code clear.

You can create a function like that:
```
function getSafeX(box: { x: number; width: number }, fraction: number, offset: number): number {
  if (fraction === 0) return box.x + offset;
  if (fraction === 1) return box.x + box.width - offset;
  return box.x + box.width * fraction;
}
```

File: e2e/tests/desktop/player/playback-seek.desktop.spec.ts [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+  await loadFile({ mainWindow, filename: MCAP_FILENAME });
+  await changeToEpochFormat(mainWindow);
+
+  const button = mainWindow.getByTestId("seek-forward-button");
+  const timestamp = mainWindow.getByTestId("PlaybackTime-text").locator("input");
+  const startTime = Number(await timestamp.inputValue());
+
+  // When
+  await button.click(); // seek forwards
+  await mainWindow.waitForTimeout(500); // wait for cursor
```
Comment (luluiz) [inline e2e/tests/desktop/player/playback-seek.desktop.spec.ts]: Maybe 500ms is much time for this kind of timeout. Did you try with 100?
If 100 is enough, please update this and and others timeouts in this file.

Comment (rodrigo-p-rodrigues) [reply e2e/tests/desktop/player/playback-seek.desktop.spec.ts]: added helper function that waits for timestamp stabilization

File: e2e/fixtures/change-to-epoch-format.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  // get date values in epoch format
+  const playerStartingTime = mainWindow.locator(`input[value="${initialTimeInUTC}"]`);
+  await playerStartingTime.hover();
+  const timestampDropdown = mainWindow.getByTestId("playback-time-display-toggle-button");
+  await timestampDropdown.click();
+
+  const newTimestampOption = mainWindow.getByTestId("playback-time-display-option-SEC");
+  await newTimestampOption.click();
+  await expect(newTimestampOption).toHaveCount(0); // wait until menu closes
+  return;
```
Comment (ctw-joao-luis) [inline e2e/fixtures/change-to-epoch-format.ts]: Why have this return if the function return type is `Promise<void>`, I think it's redundant

Comment (rodrigo-p-rodrigues) [reply e2e/fixtures/change-to-epoch-format.ts]: fixed
